.github/workflows/training-e2e.yaml

```diff
@@ -17,7 +17,7 @@ env:
   TF_VAR_aws_region: "eu-west-2"
   TF_VAR_aws_ami_owners: '["125523088429"]'
   TF_VAR_aws_ami_name: '["Fedora-Cloud-Base-*"]'
-  TF_VAR_aws_volume_size: 128
+  TF_VAR_aws_volume_size: 500
```

Comment:
Does this one also possibly need changing? https://github.com/containers/ai-lab-recipes/blob/1e39174d03409c2874c7a02d36182654d0d6f404/.github/workflows/testing_framework.yaml#L17